core/src/test/scala/com/yahoo/maha/core/RequestModelTest.scala

```diff
@@ -5819,5 +5819,29 @@ class RequestModelTest extends FunSuite with Matchers {
     println(res.failed.get.getMessage)
     res.failed.get.getMessage should startWith (s"requirement failed: ERROR_CODE:10007 (Ad Format Name) can't be used with advertiser schema in publicFact cube")
   }
+
+  test ("Dim Only query Schema required filter validation") {
```

Comment:
Can we add a test to ensure it doesn't fail when the Advertiser ID filter is present?

Reply:
working on it,

Reply:
added one positive case, found that we have many positive cases all over test classes, shouldnt be problem.